taker-frontend/src/components/History.tsx

```diff
@@ -97,7 +97,12 @@ const CfdDetails = ({ cfd }: CfdDetailsProps) => {
     });
 
     const disableCloseButton = cfd.state.getGroup() === StateGroupKey.CLOSED
-        || [StateKey.OPEN_COMMITTED, StateKey.OUTGOING_SETTLEMENT_PROPOSAL, StateKey.PENDING_CLOSE].includes(
+        || [
+            StateKey.PENDING_OPEN,
+            StateKey.OPEN_COMMITTED,
+            StateKey.OUTGOING_SETTLEMENT_PROPOSAL,
+            StateKey.PENDING_CLOSE,
+        ].includes(
             cfd.state.key,
         );
```

Comment:
Would it be easier to just say: 

```suggestion
cfd.state.getGroup() === StateGroupKey.CLOSED || !StateKey.OPEN === cfd.state.key
```